src/Header/Header.js

```diff
@@ -8,19 +8,15 @@ class Header extends Component {
 
   setAsideViewMyCards = () => {
     this.props.setAsideView("myCardList");
-    this.returnToCards()
+    this.props.setCardAreaView("myCardList");
```

Comment:
Awesome! I was going to try this to eliminate the reload so good to know that it works!